include/dali/core/tensor_shape.h

```diff
@@ -1473,6 +1473,48 @@ TensorListShape<output_ndim> sample_range(const TensorListShape<ndim> &in,
   return ret;
 }
 
+/**
+ * @brief Unfolds outer dimension of TensorShape in TensorShapeList
+ * unfold_outer_dim([[2, 1], [1, 3]]) -> [1, 1, 3]
```

Comment:
```suggestion
 * unfold_outer_dim([[2, 1], [1, 3]]) -> [[1], [1], [3]]
 * unfold_outer_dim([[4, 800, 600, 3], [2, 200, 200, 3]]) -> [[800, 600, 3], [800, 600, 3], [800, 600, 3], [800, 600, 3], [800, 600, 3], [800, 600, 3]]
```
^^ maybe one more example would help readers